pkg/operator/controller/dns_status.go

```diff
@@ -100,11 +100,9 @@ func computeDNSDegradedCondition(oldCondition *operatorv1.OperatorCondition, clu
 		want := dnsDaemonset.Status.DesiredNumberScheduled
 		have := dnsDaemonset.Status.NumberAvailable
 		numberUnavailable := want - have
-		maxUnavailableIntStr := intstr.FromInt(1)
-		if dnsDaemonset.Spec.UpdateStrategy.RollingUpdate != nil && dnsDaemonset.Spec.UpdateStrategy.RollingUpdate.MaxUnavailable != nil {
-			maxUnavailableIntStr = *dnsDaemonset.Spec.UpdateStrategy.RollingUpdate.MaxUnavailable
-		}
-		maxUnavailable, intstrErr := intstr.GetScaledValueFromIntOrPercent(&maxUnavailableIntStr, int(want), true)
+		maxUnavailableIntStr := intstr.FromString("10%")
+		maxUnavailable, _ := intstr.GetScaledValueFromIntOrPercent(&maxUnavailableIntStr, int(want), true)
```

Comment:
I'm wary of ignoring error values.  Even if an analysis of the code today determines that the error value will always be nil, it's possible that this code or the definition of `GetScaledValueFromIntOrPercent` could change in the future.

Reply:
ok